package/Resources/Translations/zh-Hans/Translation.ini

```diff
@@ -0,0 +1,1730 @@
+Client:Main:StartPositionAsANoun=起点选择; start selection
+Client:Main:StatisticBuildCount=已建造; BUILT
+Client:Main:StatisticBuildCountAvg=平均建造物体数：; Avg. number of objects built:
+Client:Main:StatisticEconomy=经济; ECONOMY
+Client:Main:StatisticEconomyAvg=平均经济：; Average economy:
+Client:Main:Statistics=统计; Statistics
+Client:Main:StatisticsAlliesAvg=平均盟友数：; Average number of allies:
+Client:Main:StatisticsEnemiesAvg=平均敌人数量：; Average number of enemies:
+Client:Main:StatisticsGamesFinished=完成的游戏：; Games finished:
+Client:Main:StatisticsGamesLengthAvg=平均游戏时长：; Average game length:
+Client:Main:StatisticsGamesLosses=失败：; Losses:
+Client:Main:StatisticsGamesStarted=游戏已开始：; Games started:
+Client:Main:StatisticsGamesWinLossRatio=胜/负比：; Win / Loss ratio:
+Client:Main:StatisticsGamesWins=胜利：; Wins:
+Client:Main:StatisticsKillLossRatio=战损比：; Kill / loss ratio:
+Client:Main:StatisticsKills=击杀; KILLS
+Client:Main:StatisticsKillsPerGame=每场击杀数：; Kills / game:
+Client:Main:StatisticsLosses=损失; LOSSES
+Client:Main:StatisticsLossesPerGame=每场损失数：; Losses / game:
+Client:Main:StatisticsName=玩家名; NAME
+Client:Main:StatisticsScore=得分; SCORE
+Client:Main:StatisticsSide=阵营; SIDE
+Client:Main:StatisticsTeam=队伍; TEAM
+Client:Main:StatisticsTotalKills=总击杀数：; Total kills:
+Client:Main:StatisticsTotalLosses=总损失数：; Total losses:
+Client:Main:StatisticsTotalTimePlayed=总游戏时间：; Total time played:
+Client:Main:StatisticsWon=胜利; WON
+Client:Main:StatusConnected=已连接; CONNECTED
+Client:Main:StatusConnecting=连接中...; CONNECTING...
+Client:Main:StatusLanMode=局域网模式; LAN ModE
+Client:Main:StatusOffline=离线; OFFLINE
+Client:Main:StatusPreparing=准备中; Preparing
+Client:Main:StayConnect=在 CnCNet 大厅外保持连接; Stay connected outside of the CnCNet lobby
+Client:Main:StillInGameNotification=无法启动游戏。玩家 {0} 仍在玩您之前开始的游戏。; Unable to launch game. Player {0} is still playing the game you started previously.
+Client:Main:SupplementalFileCopyError=无法复制补充地图文件; Unable to copy supplemental map file
+Client:Main:SupplementalFileDeleteError=无法删除补充地图文件; Unable to delete supplemental map file
+Client:Main:TeamAsANoun=队伍选择; team selection
+Client:Main:ToggleExtraIcons=切换额外图标; Toggle Extra Icons
+Client:Main:ToggleFavoriteMap=切换收藏地图; Toggle Favorite Map
+Client:Main:TooManyPlayersNotification=无法启动游戏：此地图最多只能有 {0} 名玩家。; Unable to launch game: this map cannot be played with more than {0} players.
+Client:Main:TopBarLogOut=登出; Log Out
+Client:Main:TOSButtonOK=确认; Got it
+Client:Main:TOSExplanation=使用此应用程序即表示您同意 CnCNet 服务条款以及 CnCNet 隐私政策。与隐私相关的选项可以在客户端设置中配置。; By using this application you agree to the CnCNet Terms & Conditions as well as the CnCNet Privacy Policy. Privacy-related options can be configured in the client settings.
+Client:Main:TOSMoreInfo=更多信息：; More information:
+Client:Main:TOSText=此应用程序使用 CnCNet 网络和专线隧道服务器服务，并基于此收集技术及其他必要信息。; This application makes use of CnCNet web & tunnel server services and is subject to collection of technical & other necessary information through them.
```

Comment:
I think this is the only place where `tunner server` is translated to `专线隧道` instead of `专线`。

Can change it to `专线` now for consistency. Although I don't know if players are used to `隧道` or not.